Makefile

```diff
@@ -27,7 +27,7 @@ coverage: ;
 doc: ;
 
 build:
-	docker build -t $(ARTIFACT) .
+	docker build -t $(ARTIFACT):$(VERSION) .
```

Comment:
I don't think this is actually needed, I believe Dockerhub doesn't really do anything unless we manually tag the commit. I did notice we had a `VERSION` in this makefile though and it was convenient for my local builds 🤷